### ISSUE TITLE
[First Time Experience] Need User Flow for FTE

### ISSUE THREAD

**Issue by mikehenrty** [mozilla/voice-web#45]
Here's a proposal for a simple first time experience that introduces all the features. Obviously this is still open to suggestions and/or complete rewrites.

User starts at **Homepage**:
(Text needs to be short and clear, and have call to action.)

> _Welcome to Voice Commons!
> Our goal is to create a Public Domain (CC0) database of voices in every language with any accent. We believe this data can empower universities, researchers, buinesses, non-profits (like ourselves), or anyone who is interested in Voice Recognition technology to be creators in this emerging space. You can learn more about Common Voice through our Mision Statement<link>._

Take our guided tour: <big button>
Button text could be `Lend your Voice` or `Donate` or `Try it out`

(button takes you to donate screen)

**Donate Screen**

> _insert brief description of why we need "labeled" voice clips._

(pehaps we can call out that your data won't be uploaded yet)

User clicks records their voice, and clicks `Submit`.

(here we could have some sort of nice indication they just contributed)
(submitting their voice clip takes them to the listening screen.)

**Listen Screen**

> _Explain that we need to make sure that what everyone says is correct_
> "Play this sound clip, and tell is if what they said matches the senstence"

(here users could listen to your own clip, or we could have a pre-canned example)

User selects yes or no:
Yes || no

(Whether they say yes or no, move on to registration)

> _Don't worry, what you just recorded has not been added to our database yet. First, we would like you to take a look at our Privacy Policy<link>.
> tl;dr When we publish the voice database, we will strip all personally identifying information<link to PIID info> about you. But while using this site, you may share as much or as little personal information as you'd like with other users._

> 
> _To use all the features of this site, you need to register. Again, your email will not be tied to any published voice data._
> Registration Form
> (perhaps make it clear they can still donate as Guest)

**Comment by mikehenrty** [comment]
We've had several meetings with the design studio the last two weeks, we should get a rc1 wireframes in the next day or so.

**Comment by mikehenrty** [comment]
I'm calling this done since we have this pretty well nailed down in the wireframes and latest designs.
